docs/contributing/3.0-development/rest-api.rst

```diff
@@ -80,10 +80,8 @@ Serializer Notes
   be explicit.
 
 * Serialized objects will provide thier own URL as the value of the "_href" field on the serializer.
-  If your API endpoint does not use the primary key as its lookup field (such as repository being
-  looked up by name), you will need to use a ``rest_framework.serializers.HyperlinkedIdentifyField``
-  to generate the ``_href`` field value correctly, by specifying its ``lookup_field`` and
-  ``view_name``.
+  You will need to use a ``rest_framework.serializers.HyperlinkedIdentifyField`` to generate the
+  ``_href`` field value correctly, by specifying its ``lookup_field`` and ``view_name``.
```

Comment:
by specifying its ``view_name``. If this object is referenced in the url by a field other than the pk, you will also need to specify a ``lookup_field``

Reply:
also s/correctly//